www/md_base/src/app/home/home.route.coffee

```diff
@@ -17,3 +17,33 @@ class State extends Config
             templateUrl: "views/#{name}.html"
             name: name
             url: '/'
+
+        bbSettingsServiceProvider.addSettingsGroup
+            name: 'home'
+            caption: 'Home'
+            description: 'Settings to customize the display of Home page'
+            items: [
+                type: 'bool'
+                name: 'lock_panels'
+                caption: 'Lock panels'
+                default_value: false
```

Comment:
I think this would be better to have it locked by default, and having a small button in order to unlock them directly in the home page.
ADF does it, and I find it pretty straigforward: http://sdorra.github.io/angular-dashboard-framework/#/sample/01